src/test/ui/resolve/issue-88472.stderr

```diff
@@ -0,0 +1,34 @@
+error[E0412]: cannot find type `Foo` in this scope
+  --> $DIR/issue-88472.rs:14:16
+   |
+LL |     type Bar = Foo;
+   |                ^^^ not found in this scope
+   |
+   = note: this struct exists but is inaccessible:
+           a::Foo
```

Comment:
We should use the `ImportSuggestion.did` along with `def_span` to show more verbose output:

```
note: struct `a::Foo` exists but is inacessible
  --> $DIR/issue-88472.rs:8:16
   |
LL |     struct Foo;
   |            ^^^ not accessible
```